### PR TITLE
Genfors visual fixes

### PR DESCRIPTION
**Pull request by MrHanSan**

### CODE REVIEW

File: files/static/js/genfors.js [hunk 1]
```diff
@@ -125,23 +126,26 @@ Genfors = (function () {
...
                             var value = r[key];
                             var percent;
                             if (data.question.count_blank_votes) {
                                 percent = Genfors.get_percent(value, current_votes);
+                                html += '<p><strong>' + key + '</strong>: ' + value + ' stemme' + ((value>1)?'r':'') + '</p>';
                             }
                             else {
                                 if (key == 'Blankt') {
                                     percent = Genfors.get_percent(value, current_votes);
+                                    html += '<p style="margin-top: 50px;"><strong>' + key + ' (ikke tellende)</strong>: ' + value + ' stemme' + ((value>1)?'r':'') + '</p>';
```
Comment (hernil) [inline files/static/js/genfors.js]: Inline styling :-1: 

Slap on a class to it and to it in the less!

File: files/static/js/genfors.js [hunk 2]
```diff
@@ -178,14 +182,21 @@ Genfors = (function () {
...
                         var current_votes = data.question.current_votes;
                         var html = "";
                         for (var x = 0; x < alternatives.length; x++) {
                             var key = alternatives[x];
                             var value = r[key];
                             var percent;
-                            html += '<p><strong>' + key + '</strong>: ' + value + ' stemme' + ((value>1)?'r':'') + '</p>';
+                            if (key == 'Blankt' && !data.question.count_blank_votes) {
+                                percent = Genfors.get_percent(value, current_votes);
+                                html += '<p style="margin-top: 50px;"><strong>' + key + ' (ikke tellende)</strong>: ' + value + ' stemme' + ((value>1)?'r':'') + '</p>';
```
Comment (hernil) [inline files/static/js/genfors.js]: Inline styling :-1: 

Slap on a class to it and to it in the less!

File: templates/genfors/index.html [hunk 3]
```diff
@@ -125,28 +125,30 @@
...
                                             <div class="progress-bar progress-bar-warning" id="boolean_blank_progress" role="progressbar" aria-valuenow="{{ active_question.blank_percent }}" aria-valuemin="0" aria-valuemax="100" style="width: {{ active_question.blank_percent }}%">
-                                                    {{ res.Blankt }} av {{ q.get_votes.count }} stemme{{ q.get_votes.count|pluralize:"r" }}
-                                                </div>
-                                        {% endif %}
-                                    </div>
+                                                {{ active_question.blank_percent }}%
+                                            </div>
+                                        </div>
+                                    {% else %}
+                                        <p style="margin-top: 50px"><strong>Blankt (ikke tellende)</strong>: <span id="boolean_blank">{{ res.Blankt }}</span> stemme{{ res.Blankt|pluralize:"r" }}<p />
```
Comment (hernil) [inline templates/genfors/index.html]: inline styling

**Comment by hernil** [comment]
Looks good apart from the inline styling :smile:

**Comment by MrHanSan** [comment]
ok, should be good now

File: templates/genfors/admin.html [hunk 4]
```diff
@@ -164,42 +164,34 @@
...
+                                    <td>
+                                    {% with question.get_admin_results as results %}
+                                        Resultat: <strong>{{ question.get_leader }}</strong><br />
+                                        {% if results.valid %}
+                                             <span class="label label-success">Flertall</span><br /><br />
+                                        {% else %}
+                                             <span class="label label-danger">Ikke flertall</span><br /><br />
+                                        {% endif %}
+                                        {% for alt, votes in results.data.items %}
+                                             <span style="white-space: nowrap"><strong>{{ alt }}</strong>: {{ votes }} stemme{{ votes|pluralize:"r"}}.</span><br />
```
Comment (OptimusCrime) [inline templates/genfors/admin.html]: inline styling here

File: templates/genfors/admin.html [hunk 5]
```diff
@@ -164,42 +164,34 @@
...
-                                                </tr>
-                                                {% empty %}
-                                                <tr>
-                                                    <td colspan="3">Ingen avsluttede saker.</td>
-                                                </tr>
-                                                {% endfor %}
-                                            </table>
-                                        </div>
+                                    <td>{{ question.description }}</td>
+                                    <td style="white-space: nowrap">{% if question.majority_type == 0 %}Alminnelig{% else %}Kvalitativt{% endif %}</td>
```
Comment (OptimusCrime) [inline templates/genfors/admin.html]: inline styling here

**Comment by MrHanSan** [comment]
ok, should be good now, anything else?

**Pull request merged**